frontend/src/components/StudentProfile/AuditHistory/CourseChangeHistory.vue

```diff
@@ -0,0 +1,145 @@
+          <v-row dense>
+            <v-col
+                v-for="field in courseExamFields"
+                :key="field.key"
+                class="text-center"
+                :title="field.description"
+            >
+              <div class="text-caption font-weight-bold">{{ field.label }}</div>
+              <div class="text-body-2">{{ item.courseExam[field.key] ?? '' }}</div>
+            </v-col>
+          </v-row>
+        </td>
+      </tr>
+    </template>
+    <template
+        v-slot:item.data-table-expand="{
+                item,
+                internalItem,
+                toggleExpand,
+                isExpanded,
+              }"
+    >
+      <td v-if="!!item.courseExam">
+        <v-btn
+            variant="text"
+            density="comfortable"
+            size="small"
+            @click="toggleExpand(internalItem)"
+            class="v-data-table__expand-icon"
+            :class="{ 'v-data-table__expand-icon--active': isExpanded }"
+            :icon="
+                    isExpanded(internalItem)
+                      ? 'mdi-chevron-down'
+                      : 'mdi-chevron-right'
+                  "
+        >
+        </v-btn>
+      </td>
+    </template>
+    <template v-slot:item.activityCode="{ item }">
+      <i>{{ item.activityCode }}</i> -
+      {{ item.activityDescription }}
+    </template>
+    <template v-slot:item.updateDate="{ item }">
+      {{ $filters.formatTime(item.updateDate) }}
+    </template>
+  </v-data-table>
+</template>
+
+<script>
+
+import {mapActions, mapState} from "pinia";
+import {useStudentStore} from "@/store/modules/student";
+
+export default {
+  props: {},
+  computed: {
+    ...mapState(useStudentStore, {
+      studentId: "getStudentId",
+      courseHistory: "getStudentCourseAuditHistory"
+    }),
+  },
+  data: function () {
+    return {
+      courseChangeHistory: [],
+      courseFields: [
+        { key: "data-table-expand", title: "" },
+        { key: "updateDate", title: "Date", width: "115px" },
+        { key: "activityCode", title: "Change" },
+        { key: "updateUser", title: "Update User", sortable: true },
+        { key: "courseDetails.courseCode", title: "Code" },
+        { key: "courseDetails.courseLevel", title: "Level" },
+        { key: "courseSession", title: "Session" },
+        { key: "interimPercent", title: "%", description: "Interim Percent" },
+        { key: "interimLetterGrade", title: "LG", description: "Interim Letter Grade" },
+        { key: "finalPercent", title: "%", description: "Final Percent" },
+        { key: "finalLetterGrade", title: "LG", description: "Final Letter Grade" },
+        { key: "credits", title: "Credits" },
+        { key: "equivOrChallenge", title: "EC", description: "Equivalent or Challenge" },
+        { key: "fineArtsAppliedSkills", title: "AS", description: "Fine Arts or Applied Skills" },
+        { key: "customizedCourseName", title: "Course Title" },
```

Comment:
Should we rename the title to "Custom Course Title" so it's clear it's for the custom course title and not the title of the course you get from the code and level?